tests/blobstore_common_tests.py

```diff
@@ -155,7 +164,7 @@ def _testListV2Continuation(self, page_size, break_size):
             self.test_fixtures_bucket,
             "testList/prefix",
             token=blobiter.token,
-            start_after_key=items1[-1],
+            start_after_key=items1[-1][0],
```

Comment:
this is a bit hard to read now.  not sure if you want items1 only to include the keys.